pkg/databind/internal/secrets/cyberarkcli_test.go

```diff
@@ -0,0 +1,54 @@
+package secrets
+
+import (
+	"fmt"
+	"github.com/newrelic/infrastructure-agent/pkg/databind/pkg/data"
+	"os"
+	"os/exec"
+	"testing"
+)
+
+func fakeExecCommand(command string, args ...string) *exec.Cmd {
+	cs := []string{"-test.run=TestHelperProcess", "--", command}
+	cs = append(cs, args...)
+	cmd := exec.Command(os.Args[0], cs...)
+	cmd.Env = []string{"GO_WANT_HELPER_PROCESS=1"}
+	return cmd
+}
+
+func TestHelperProcess(t *testing.T) {
```

Comment:
I'm concerned that you have an `os.Exit(0)` here and wondering is it needed? What are you trying to achieve here? The problem I have with this is that `os.Exit(0)` can hide failures in other tests if we aren't careful. Where you use the custom command can you not replace it with `exit 0` or `false` instead?

Reply:
This is actually a spawned sub process. The test causes Go to invoke  another test environment and it's in this other environment that `os.exit` is invoked, not the main Go test environment.  We want this test to simulate a successful call to the CLI.